Observer/CheckoutObserver.php

```diff
@@ -92,6 +92,18 @@ public function afterSaveOrder(Varien_Event_Observer $event)
         }
     }
 
+    public function onShippingUpdate(Varien_Event_Observer $event)
```

Comment:
For clarity, this method should be titled something more descriptive such as "updateBoldOrderShippingInformation" or something similar.

Reply:
Done.